CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 Changelog
 =========
 
+1.5.0 (2019-08-23)
+------------------
+
+* feature: return cached vms list for rpc and panel
```

Comment:
be more verbose, it's not a ordinary cache - the data is still the latest possible across cycles/instances

Reply:
Fixed.